packages/react-ui/src/widgets/CategoryWidgetUI.js

```diff
@@ -510,11 +511,7 @@ function CategoryWidgetUI(props) {
                 />
               ))
             ) : data.length === 0 && !isLoading ? (
-              <Alert severity='warning'>
-                <AlertTitle>No data available</AlertTitle>
-                There are no results for the combination of filters applied to your data.
-                Try tweaking your filters, or zoom and pan the map to adjust the Map View.
-              </Alert>
+              <NoDataAlert />
```

Comment:
If you create another PR for this mini-feature, then this should be included there

Reply:
Yes.